modules/cluster/locals.tf

```diff
@@ -39,6 +39,7 @@ data:
       groups:
         - system:bootstrappers
         - system:nodes
+${var.aws_auth}
```

Comment:
Does this work when the indentation is not 4 spaces?

Reply:
@vlesierse No, it does not work. Tabs are generally forbidden in YAML (http://yaml.org/faq.html). The authentication is a very important part of the workflow and we should give the possibility to use this feature. I still think that we should go with this.